### DISCUSSION TITLE
Raise a warning in github workflow

### DISCUSSION THREAD

**Discussion by javixeneize** [community/community#27123] [Actions]
<p>Hi</p>
<p>I there any way to mark a job as unstable? I have added a step in my workflow to create an annotation, and yes, it is displayed in the summary, but the job is still marked as green as the workflow ended successfully.</p>
<pre><code>  - name: Generate Warning Annotations
    run: |
      echo "::warning ::Review needed"
</code></pre>
<p>How can i mark the job as unstable?</p>
<p>Thanks</p>

**Comment by mickeygousset** [comment]
<p>I’ll be honest, I can’t seem to find a good way to do this.</p>

**Comment by javixeneize** [comment]
<p>Thanks for the answer. Do you consider it as a good addition? I can open an issue (don’t know where) to suggest that improvement</p>
<p>Thanks</p>

**Comment by mickeygousset** [comment, marked as answer]
<p>I’d go here and submit the feedback.  <a href="https://support.github.com/contact/feedback" class="inline-onebox">Share feedback - GitHub Support</a></p>
<p>I’ll also enter your suggestion into an internal system as well.</p>
